src/scripts/install.py

```diff
@@ -363,20 +382,26 @@ def setup_authentication(jira_password, region, pk_passphrase=None):
     return private_key, pk_passphrase, jira_password, rsa_public_key
 
 
-def gen_envs(jira_user, jira_project, jira_url, jira_password, account, region,
+def gen_envs(jira_user, jira_project, jira_url, jira_password, account, region, host, port, protocol,
              private_key, pk_passphrase, aws_key, aws_secret, **x):
+    # for the .envs file being written, we redesigned the format,
+    # so that currently, host become the TOP argument
     vars = [
-        ('SNOWFLAKE_ACCOUNT', account),
-        ('SA_USER', USER),
-        ('SA_ROLE', ROLE),
-        ('SA_DATABASE', DATABASE),
-        ('SA_WAREHOUSE', WAREHOUSE),
-        ('REGION', region or 'us-west-2'),
+        ('SNOWFLAKE_HOST', host),
+        ('SH_ACCOUNT', account),
+        ('SH_USER', USER),
+        ('SH_ROLE', ROLE),
+        ('SH_DATABASE', DATABASE),
+        ('SH_WAREHOUSE', WAREHOUSE),
+        ('SH_REGION', region or 'us-west-2'),
+        ('SH_PORT', port),
+        ('SH_PROTOCOL', protocol),
 
         ('PRIVATE_KEY', b64encode(private_key).decode("utf-8")),
         ('PRIVATE_KEY_PASSWORD', pk_passphrase),
     ]
 
+
```

Comment:
please avoid adding extra newlines. also, please set up your text editor to warn you about violating PEP-8.